plugins/catalog-backend/src/service/router.ts

```diff
@@ -88,7 +88,8 @@ export async function createRouter(
   if (higherOrderOperation) {
     router.post('/locations', async (req, res) => {
```

Comment:
My first draft also added dryRun to `POST /entities` but I don't need it right now and I think this way it is sufficient.